xcm/xcm-builder/src/currency_adapter.rs

```diff
@@ -52,22 +52,26 @@ impl From<Error> for XcmError {
 ///
 /// # Example
 /// ```
-/// use frame_support::parameter_types;
+/// use parity_scale_codec::Decode;
+/// use frame_support::{parameter_types, PalletId};
+/// use sp_runtime::traits::{AccountIdConversion, TrailingZeroInput};
 /// use xcm::latest::prelude::*;
-/// use xcm_builder::{ParentIsDefault, CurrencyAdapter, IsConcrete};
+/// use xcm_builder::{ParentIs, CurrencyAdapter, IsConcrete};
 ///
 /// /// Our chain's account id.
 /// type AccountId = sp_runtime::AccountId32;
 ///
 /// /// Our relay chain's location.
 /// parameter_types! {
 ///     pub RelayChain: MultiLocation = Parent.into();
-///     pub CheckingAccount: AccountId = Default::default();
+///     pub CheckingAccount: AccountId =
+///         AccountId::decode(&mut TrailingZeroInput::zeroes()).unwrap();
```

Comment:
Do we really want to have such "hacks" in the docs?

Reply:
I don't know what else you can put into here, you want me to do the same thing with `PalletId` instead?

Reply:
```suggestion
///     pub CheckingAccount: AccountId = PalletId(*b"checking").into_account();
```